lib/morse_translator.rb

```diff
@@ -38,6 +38,41 @@ def initialize
                     "9" => "----.",
                     "0" => "-----",
                     " " => " "}
+    @morse_string = ""
```

Comment:
I'm not sure about having this as an instance variable - it seems like it should just be the return value of the #translate method.